src/messages.ts

```diff
@@ -0,0 +1,73 @@
+import {BeaconBlockBody, BeaconBlockHeader, BeaconBlockRoot} from "./types";
+
+export interface RpcBody {
+  method_id: number;
+  id: 1;
+  body: requests;
+}
+
+type requests = Hello | Goodbye | GetStatus | GetBlockRoots | BlockRoots | GetBlockHeaders | BlockHeaders | GetBlockBodies | BlockBodies;
```

Comment:
Might be insignificant to some, but I think `requests` should be changed to `request`, as the plural reads as if it's a list type.

Reply:
good point!